.github/workflows/publish.yml

```
@@ -194,4 +192,17 @@ jobs:
           file: ${{ steps.build_msi.outputs.msi }}
           asset_name: compose-generator_${{ matrix.config.label }}_setup.msi
           tag: ${{ github.ref }}
-          
\ No newline at end of file
+
+  deploy-win-installers:
+    name: Deploy installers
+    needs: build-win-installers
+    runs-on: windows-latest
+
+    steps:
+      - name: Release to WinGet
+        uses: vedantmgoyal2009/winget-releaser@v2
```

```suggestion
        uses: vedantmgoyal9/winget-releaser@main
```

Use the latest version of the action.